packages/frontend/src/components/LightdashVisualization/VisualizationProvider.tsx

```diff
@@ -75,9 +76,11 @@ type VisualizationContext = {
     setChartType: (value: ChartType) => void;
     setPivotDimensions: (value: string[] | undefined) => void;
 
-    getSeriesColor: (seriesLike: SeriesLike) => string;
+    getSeriesColor: (
+        seriesLike: SeriesLike,
+        metadata?: Record<string, SeriesMetadata>,
+    ) => string;
```

Comment:
```suggestion
    getSeriesColor: (
        seriesLike: SeriesLike,
    ) => string;
```

I haven't seen this argument getting used/passed anywhere - also the function on line 255 doesn't seem to take it into account

Reply:
yes, you're right, good catch. This was probably a leftover for my tests